src/test/scala/uk/gov/hmcts/ccd/util/PerformanceTestsConfig.scala

```diff
@@ -17,6 +17,16 @@ trait PerformanceTestsConfig {
   val CaseDefinitionUrl = config.getString("caseDefinitionUrl")
   val MaxSimulationDuration = config.getInt("maxSimulationDurationMinutes")
   val RequestRateSec = Option(System.getProperty("request.rate")).map(_.toDouble)
+  val minThinkTime = config.getInt("minthinktime")
```

Comment:
in scala constants starts with uppercase. can you change all of the new constants to have first letter uppercase please?

Reply:
Done